release-notes/DataAggregator/DataAggregator_2.0.0.md

```diff
@@ -0,0 +1,17 @@
+---
+uid: DataAggregator_2.0.0
+---
+
+# Data Aggregator 2.0.0
+
+## New features
+
+-
```

Comment:
Not sure if there should be an empty section here.

Reply:
Since no other RNs will be added to this release, I'll remove this.